build-docs.sh

```diff
@@ -6,6 +6,7 @@ set -e
 # Gather expected arguments.
 CODE_DIR=$1
 DOCS_DIR=$2
+DOCS_RAW_DIR=$3
```

Comment:
this only needs to be two args as DOCS_DIR would end up unused so the value being passed in $3 can be passed in $2 instead.